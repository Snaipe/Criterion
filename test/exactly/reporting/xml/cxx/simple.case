[act]

% simple.cc.bin --xml

[assert]

stdout
  -transformed-by ( replace_version | replace_times )
  equals -contents-of expected/simple.txt
