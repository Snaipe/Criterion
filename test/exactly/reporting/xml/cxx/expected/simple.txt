<?xml version="1.0" encoding="UTF-8"?>
<!-- Tests compiled with Criterion <VERSION> -->
<testsuites name="Criterion Tests" tests="2" failures="1" errors="0" disabled="0">
  <testsuite name="misc" tests="2" failures="1" errors="0" disabled="0" skipped="0" time="<TIME>">
    <testcase name="passing" assertions="1" status="PASSED" time="<TIME>">
    </testcase>
    <testcase name="failing" assertions="1" status="FAILED" time="<TIME>">
      <failure type="assert" message="1 assertion(s) failed."><![CDATA[simple.cc:4: The expression 0 is false.]]>&#10;</failure>
    </testcase>
  </testsuite>
</testsuites>
